### ISSUE TITLE
train.py:692 cites probes/t57_outlier.py as "tested rather than assumed", but that probe only prints — it enforces no threshold (high)

### ISSUE THREAD

**Issue by cklxx** [acupof-ai/aupai#559]
## Finding (read-only audit, 2026-09-19, genA)

`train.py:692` says, of per-tensor fp8 scaling:

> Per-tensor scaling is robust to an outlier, tested rather than assumed (`probes/t57_outlier.py`, …)

But `probes/t57_outlier.py` (33 lines) has **zero `assert`, zero `raise`, zero conditional exit**. It plants an outlier, computes a scale, and only `print(...)`s the numbers:

```
scale = a.detach().abs().amax() / 448.0
print(f"outlier {outlier:7.1f} -> a_scale {scale:.4f} | ordinary rows: ...")
```

A printed number does not fail on a regression. This is the most dangerous vacuous-green in the audit because the non-enforcement is asserted **inside production code comments as if it were a tested guarantee** — a future change that breaks per-tensor scaling will not turn anything red, yet the source still claims it is "tested". (Cross-refs the broader probe-hygiene issue being filed alongside.)

## Fix — either is acceptable
1. **Make the probe a test:** add an assertion/threshold (e.g. the capped ordinary-row logits stay within tolerance of the no-outlier case while the outlier row is bounded), exiting nonzero when it regresses — then register it in the appropriate tier.
2. **Or correct the claim:** change the comment from "tested rather than assumed" to "measured once (…); threshold is not enforced by any test", so production code does not cite a print-only probe as verification.

Option 1 is preferred if the property is load-bearing; option 2 at minimum removes the false guarantee.

Evidence: `grep -cE "assert|raise|sys.exit" probes/t57_outlier.py` = 0/0/0; only a print of the scale.

**Comment by cklxx** [comment]
Related: probe hygiene/marking for t57_outlier and the other manual-only probes in #560; orphan-gate pattern in #558.
